dask_expr/io/io.py

```diff
@@ -147,6 +148,58 @@ def _tune_up(self, parent):
         return
 
 
+class FusedParquetIO(FusedIO):
+    _parameters = ["_expr"]
+
+    @functools.cached_property
+    def _name(self):
+        return (
+            funcname(type(self.operand("_expr"))).lower()
+            + "-fused-parq-"
+            + _tokenize_deterministic(*self.operands)
+        )
+
+    @staticmethod
+    def _load_multiple_files(
+        frag_filters,
+        columns,
+        schema,
+        *to_pandas_args,
+    ):
+        from dask_expr.io.parquet import ReadParquetPyarrowFS
+
+        tables = (
+            ReadParquetPyarrowFS._fragment_to_table(
+                frag,
+                filter,
+                columns,
+                schema,
+            )
+            for frag, filter in frag_filters
+        )
+        table = pa.concat_tables(tables, promote_options="permissive")
+        return ReadParquetPyarrowFS._table_to_pandas(table, *to_pandas_args)
+
+    def _task(self, index: int):
+        expr = self.operand("_expr")
+        bucket = self._fusion_buckets[index]
+        fragments_filters = []
+        assert bucket
+        to_pandas_args = ()
+        for i in bucket:
+            _, frag_to_table, *to_pandas_args = expr._filtered_task(i)
+            fragments_filters.append((frag_to_table[1], frag_to_table[2]))
+            columns = frag_to_table[3]
```

Comment:
The fact that this get's overwritten on every pass is a little bit confusing, but can easily be a follow up